src/FlubuCore/Infrastructure/FlubuConsoleLogger.cs

```diff
@@ -59,7 +58,7 @@ public static Color Color
         {
             private get
             {
-                _useColor = false;
+                _useColor = true;
```

Comment:
**Note:** I'm not sure why it was assigned to `false`. I could be wrong here.

Reply:
When the color is accessed by the FlubuConsoleLogger it shoud be set to false so logger doesn't use the color again. 

```c#
    protected override void ConfigureTargets(ITaskContext context)
    {
        context.LogInfo("Some message", Color.Red);
        context.LogInfo("Another message");
    }
```
if you don't set _useColor to false in getter both messages will be logged with color red. So this change should be reverted.

Reply:
> if you don't set _useColor to false in getter both messages will be logged with color red. So this change should be reverted.

Got it. I'll take a further look. Thanks!